uvm/impl/com/untangle/uvm/SettingsManagerImpl.java

```diff
@@ -551,17 +582,11 @@ private void _saveCommit( String fileName, String outputFileName, boolean saveVe
             
                 String username = null;
                 String hostname = null;
-                if((UvmContextImpl.getInstance().threadRequest() != null) &&
-                   (UvmContextImpl.getInstance().threadRequest().get() != null)){
-                    username = UvmContextImpl.getInstance().threadRequest().get().getRemoteUser();
-                    HostnameLookup reports = (HostnameLookup) UvmContextFactory.context().appManager().app("reports");
-                    hostname = UvmContextImpl.getInstance().threadRequest().get().getRemoteAddr();
-                    if( reports != null && hostname != null){
-                        hostname = reports.lookupHostname(InetAddress.getByName(UvmContextImpl.getInstance().threadRequest().get().getRemoteAddr()));
-                    }
-                    if( hostname == null ){
-                        hostname = UvmContextImpl.getInstance().threadRequest().get().getRemoteAddr();
-                    }
+                String userHostNameInfo = getUserAndHostNameInfo("reports");
+                String infoArray[] = userHostNameInfo.split(",");
```

Comment:
Since `getUserAndHostNameInfo()` can return null, may need to add null check on `userHostNameInfo` variable

Reply:
Done